cli/bench_command.go

```diff
@@ -63,31 +66,32 @@ const (
 func configureBenchCommand(app commandHost) {
 	c := &benchCmd{}
 	bench := app.Command("bench", "Benchmark utility").Action(c.bench)
-	bench.Arg("subject", "Subject to use for testing").Required().StringVar(&c.subject)
+	bench.Arg("subject", "Subject to use for the benchmark").Required().StringVar(&c.subject)
 	bench.Flag("pub", "Number of concurrent publishers").Default("0").IntVar(&c.numPubs)
 	bench.Flag("sub", "Number of concurrent subscribers").Default("0").IntVar(&c.numSubs)
+	bench.Flag("js", "Use JetStream streaming").Default("false").BoolVar(&c.js)
```

Comment:
“JetStream Streaming” is that really a way we are referring to it?  I think in docs it’s mostly just JetStream so reads a bit odd here

Reply:
I think it's an attempt at distinguishing --js (streaming benchmark) vs --kv (kv benchmarking) buy yeah it ends up reading a bit odd so cleaning it up now.